### PR TITLE
Themes: Indigo Updates for Calendar, Grid, Combo, and Select

### PR DESCRIPTION
**Pull request by simeonoff**
Closes [#14407](https://github.com/IgniteUI/igniteui-angular/issues/14407)

### Additional information (check all that apply):
 - [ ] Bug fix
 - [ ] New functionality
 - [ ] Documentation
 - [ ] Demos
 - [ ] CI/CD

### Checklist:
 - [ ] All relevant tags have been applied to this PR
 - [ ] This PR includes unit tests covering all the new code ([test guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Test-implementation-guidelines-for-Ignite-UI-for-Angular))
 - [ ] This PR includes API docs for newly added methods/properties ([api docs guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Documentation-Guidelines))
 - [ ] This PR includes `feature/README.MD` updates for the feature docs
 - [ ] This PR includes general feature table updates in the root `README.MD`
 - [ ] This PR includes `CHANGELOG.MD` updates for newly added functionality
 - [ ] This PR contains breaking changes
 - [ ] This PR includes `ng update` migrations for the breaking changes ([migrations guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Update-Migrations))
 - [ ] This PR includes behavioral changes and the feature specification has been updated with them

### CODE REVIEW

**Comment by AnjiManova** [comment]
**Calendar**

**Please wait we are currently reviewing and more comments will be added here for the Light Mode**

Light Mode

- On hover, the weekend days should be like the others and should use grays.800
image

- Disabled Active date should be with grays.600 and 38% opacity
image

- Current & Special on Hover the text should be 700 the inner border 600
image

- Current & Special -> Focused looks like this it is triggered from the inspect mode but they should be as the hover state ^^
image

**Comment by yradoeva** [comment]
**Calendar**

**Please wait we are currently reviewing and more comments will be added here for the Dark Mode

Dark Mode**

-

**Comment by AnjiManova** [comment]
### Calendar

**Light Mode**

1. **Active: Disabled** - should be grays.600 with 38% opacity
2. **Current + Special: Hover** - text color should be primary.700
3. **Current + Special: Focused** (triggered with class "active") - text color should be primary.700
4. **Selected + Special + Current: Idle** - :after should be white with 80% opacity
5. **Selected + Special + Current: Hover** - :after should be white with 80% opacity
6. **Selected + Special + Current: Disabled** (triggered with class "disabled") - :after should be white with 80% opacity
7. **Range/Active: Disabled** - text color should be grays.600
8. **Range/Current: Idle** - background should be primary.50 with 50% opacity
9. **Range/Current: Disabled** (triggered with class "disabled") - the background should be primary.50 with 50% opacity
10. **Range/Special: Idle** - background should be transparent
11. **Range/Special: Hover** - :after should be primary.600
12. **Range/Special: Disabled** (triggered with class "disabled") - the background should be transparent
13. **Range/Current + Special: Idle** - the background color should be primary.50 with 50% opacity
14. **Range/Current + Special: Hover** - text color should be primary.700
15. **Range/Current + Special: Focused** (triggered with class "active") - text color should be primary.700 and the :after should be primary.600
16. **Range/Current + Special: Disabled** - background color should be primary.50 with 50% opacity

**Comment by AnjiManova** [comment]
### Calendar

Light Mode

1. **Special: Hover** - text color should be primary.600
2. **Special: Focused** (triggered with class "active") - text color should be primary.600
3. **Range/Active: Disabled** (triggered with class "disabled") - text color should be grays.600 with 38%
4. **Range/Special: Hover** - text color should be primary.600, :after should be primary.600 as well
5. **Range/Special: Focused** (triggered with class "active") - text color should be primary.600
6. **Range/Special: Disabled** (triggered with class "disabled") - text color should be primary.500
7. **Range/Current + Special: Idle** - the background should be primary.50 with 50% opacity
8. **Range/Current + Special: Focused** (triggered with class "active") - :after should be primary.600
9. **Range/Current + Special: Disabled** (triggered with class "disabled") - the background should be primary.50 with 50% opacity

**Comment by yradoeva** [comment]
Calendar

**Dark mode** (only date states, month & year picker are not yet reviewed)

1. **Active - Idle - _Weekends only!_** - text color should be grays.900 (not grays.800).
2. **Active - Disabled:** text color should be grays.400 0,38 (not grays.600 0,38)
-date-disabled-foreground

3. **Current - Hover:** text color should be white (not grays.900)
--date-current-hover-foreground

4. **Range: Active - Disabled:** text color should be grays.400 with 38% opacity (not grays.900 with 38% opacity)
Pasted Graphic 34

5. **Range: Current - Idle:** text color should be grays.900 (not grays.800)
Pasted Graphic 35

6. **Range: Current - Hover / Focused:** text color should be white (not grays.900)
Pasted Graphic 38

7. **Range: Current - disabled:** text color should be grays.900 with 38% opacity (not grays.800 with 38% opacity)
Pasted Graphic 41

8. **Range: Special - Hover:** border color should be primary.50 (not primary.100)
Pasted Graphic 43




_The ones below need to be verified by Andie if the text color should be white, or the same as for the Special dates, in the respective states. @andiesm813 please check the text colors for:_

9. **Current + Special - Idle:** text color should be white (not primary.100)
Pasted Graphic 29

10. **Current + Special - Hover / Focused:** text color should be white (not primary.50)
Pasted Graphic 31

11. **Current + Special - Disabled:** text color should be white with 38% opacity (not primary.100 with 38% opacity)
Pasted Graphic 32

12. **Range: Current + Special - Idle:** text color should be white (not primary.100)
Pasted Graphic 45

13. **Range: Current + Special - Hover:** text color should be white (not primary.50)
Pasted Graphic 51

14. **Range: Current + Special - Focused:** text color should be white (not primary.50)
Inner border (::after) should be primary.50 (not primary.100)
Pasted Graphic 50

15. **Range: Current + Special - Disabled:** text color should be white with 38% opacity (not primary.100 with 38% opacity)
Pasted Graphic 52

**Comment by andiesm813** [comment]
> ### Calendar
> Light Mode
> 
> 1. **Special: Hover** - text color should be primary.600
> 2. **Special: Focused** (triggered with class "active") - text color should be primary.600
> 3. **Range/Active: Disabled** (triggered with class "disabled") - text color should be grays.600 with 38%
> 4. **Range/Special: Hover** - text color should be primary.600, :after should be primary.600 as well
> 5. **Range/Special: Focused** (triggered with class "active") - text color should be primary.600
> 6. **Range/Special: Disabled** (triggered with class "disabled") - text color should be primary.500
> 7. **Range/Current + Special: Idle** - the background should be primary.50 with 50% opacity
> 8. **Range/Current + Special: Focused** (triggered with class "active") - :after should be primary.600
> 9. **Range/Current + Special: Disabled** (triggered with class "disabled") - the background should be primary.50 with 50% opacity
> 
> **Year and Month picker are not reviewed** the comments are related only to the dates' states in the Calendar


> Calendar
> 
> **Dark mode** (only date states, month & year picker are not yet reviewed)
> 
> 1. **Active - Idle - _Weekends only!_** - text color should be grays.900 (not grays.800).
> 2. **Active - Disabled:** text color should be grays.400 0,38 (not grays.600 0,38)
>    -date-disabled-foreground
> 3. **Current - Hover:** text color should be white (not grays.900)
>    --date-current-hover-foreground
> 4. **Range: Active - Disabled:** text color should be grays.400 with 38% opacity (not grays.900 with 38% opacity)
>    Pasted Graphic 34
> 5. **Range: Current - Idle:** text color should be grays.900 (not grays.800)
>    Pasted Graphic 35
> 6. **Range: Current - Hover / Focused:** text color should be white (not grays.900)
>    Pasted Graphic 38
> 7. **Range: Current - disabled:** text color should be grays.900 with 38% opacity (not grays.800 with 38% opacity)
>    Pasted Graphic 41
> 8. **Range: Special - Hover:** border color should be primary.50 (not primary.100)
>    Pasted Graphic 43
> 
> _The ones below need to be verified by Andie if the text color should be white, or the same as for the Special dates, in the respective states. @andiesm813 please check the text colors for:_
> 
> 9. **Current + Special - Idle:** text color should be white (not primary.100)
>    Pasted Graphic 29
> 10. **Current + Special - Hover / Focused:** text color should be white (not primary.50)
>     Pasted Graphic 31
> 11. **Current + Special - Disabled:** text color should be white with 38% opacity (not primary.100 with 38% opacity)
>     Pasted Graphic 32
> 12. **Range: Current + Special - Idle:** text color should be white (not primary.100)
>     Pasted Graphic 45
> 13. **Range: Current + Special - Hover:** text color should be white (not primary.50)
>     Pasted Graphic 51
> 14. **Range: Current + Special - Focused:** text color should be white (not primary.50)
>     Inner border (::after) should be primary.50 (not primary.100)
>     Pasted Graphic 50
> 15. **Range: Current + Special - Disabled:** text color should be white with 38% opacity (not primary.100 with 38% opacity)
>     Pasted Graphic 52

@yradoeva @desig9stein Dani's comments are ok. With the changes i made today (the changes in the hover foregrounds of the special / range special / current + special / range current + special in the dark theme), they should be inline with these comments above.

**Comment by yradoeva** [comment]
> Calendar
> 
> **Dark mode** (only date states, month & year picker are not yet reviewed)
> 
> 1. **Active - Idle - _Weekends only!_** - text color should be grays.900 (not grays.800).
> 2. **Active - Disabled:** text color should be grays.400 0,38 (not grays.600 0,38)
>    -date-disabled-foreground
> 3. **Current - Hover:** text color should be white (not grays.900)
>    --date-current-hover-foreground
> 4. **Range: Active - Disabled:** text color should be grays.400 with 38% opacity (not grays.900 with 38% opacity)
>    Pasted Graphic 34
> 5. **Range: Current - Idle:** text color should be grays.900 (not grays.800)
>    Pasted Graphic 35
> 6. **Range: Current - Hover / Focused:** text color should be white (not grays.900)
>    Pasted Graphic 38
> 7. **Range: Current - disabled:** text color should be grays.900 with 38% opacity (not grays.800 with 38% opacity)
>    Pasted Graphic 41
> 8. **Range: Special - Hover:** border color should be primary.50 (not primary.100)
>    Pasted Graphic 43
> 
> _The ones below need to be verified by Andie if the text color should be white, or the same as for the Special dates, in the respective states. @andiesm813 please check the text colors for:_
> 
> 9. **Current + Special - Idle:** text color should be white (not primary.100)
>    Pasted Graphic 29
> 10. **Current + Special - Hover / Focused:** text color should be white (not primary.50)
>     Pasted Graphic 31
> 11. **Current + Special - Disabled:** text color should be white with 38% opacity (not primary.100 with 38% opacity)
>     Pasted Graphic 32
> 12. **Range: Current + Special - Idle:** text color should be white (not primary.100)
>     Pasted Graphic 45
> 13. **Range: Current + Special - Hover:** text color should be white (not primary.50)
>     Pasted Graphic 51
> 14. **Range: Current + Special - Focused:** text color should be white (not primary.50)
>     Inner border (::after) should be primary.50 (not primary.100)
>     Pasted Graphic 50
> 15. **Range: Current + Special - Disabled:** text color should be white with 38% opacity (not primary.100 with 38% opacity)
>     Pasted Graphic 52

@desig9stein comments 9, 11, 12, 15 might be irrelevant. Please wait to clarify them before implementing.

**Comment by yradoeva** [comment]
**Calendar**
**Dark mode**

**Year/Month picker**

1. **Year/Month picker - Idle:** text should be white 80% (not grays.700)
August 2024

2. **Picker Arrow - Idle:** color should be white 80% (not grays.600)
Pasted Graphic 59

3. **Primary and Secondary Title:** text color should be white 90% (not grays.800)
Select Date

4. **Active - Hover**: text color should stay the same as in idle - grays.900 (not grays.800)
February

5. **Current - Hover**: text color should stay the same as in idle - grays.900 (not grays.800)
Pasted Graphic 67

**Comment by yradoeva** [comment]
**Calendar**

**Dark mode** - date states - updated with the latest in Figma

1. **Active - Idle - _Weekends only!_** - text color should be grays.900 (not grays.800).
2. **Active - Disabled:** text color should be grays.400 0,38 (not grays.600 0,38)
image

3. **Current - Hover:** text color should be white (not grays.900)
image

4. **Range: Active - Disabled:** text color should be grays.400 with 38% opacity (not grays.900 with 38% opacity)
Pasted Graphic 34

5. **Range: Current - Idle:** text color should be grays.900 (not grays.800)
Pasted Graphic 35

6. **Range: Current - Hover / Focused:** text color should be white (not grays.900)
Pasted Graphic 38

7. **Range: Current - disabled:** text color should be grays.900 with 38% opacity (not grays.800 with 38% opacity)
Pasted Graphic 41

8. **Range: Special - Hover:** border color should be primary.50 (not primary.100)
Pasted Graphic 43

9. **Range: Current + Special - Focused:** inner border (::after) should be primary.50 (not primary.100)
Pasted Graphic 50

**Comment by imincheva** [comment]
### Combo & Select

- Label should be linked with caption text style
- Gap between clear icon and toggle icon should be 4px in total on the Combo in Medium size. Currently, both the clear and toggle have 4px padding on both sides, which results in a bigger gap. The same gap should be applied on the Select, but instead between the suffix icon and toggle icon. @andiesm813 can you please specify what gap needs to be applied for Large and Small, because we don't have them in the kit.

**Light Mode:**
- Toggle icon should be grays.600 in Idle state. (The same var is used in hover state, but it probably shouldn't be like this, because on hover the toggle should be grays.700 like it's currently implemented.)
- Toggle icon should be grays.600 in Filled state.
- Toggle icon should be grays.700 in Filled & Hover state.
- Toggle icon should be grays.900 20% in Disabled and Filled & Disabled states.

**Dark Mode:**
- Toggle icon should be white 60% in Idle state.
- Toggle icon should be white 80% in Hover state.
- Toggle icon should be white 60% in Filled state.
- Toggle icon should be white 80% in Filled & Hover state.
- Toggle icon should be white 20% in Disabled state.

**Comment by simeonoff** [comment]
> **Dark Mode:**
> 
> * Toggle icon should be white 60% in Idle state.
> * Toggle icon should be white 80% in Hover state.
> * Toggle icon should be white 60% in Filled state.
> * Toggle icon should be white 80% in Filled & Hover state.
> * Toggle icon should be white 20% in Disabled state.

White? This directly collides with the philosophy we have for the theming that colors listed in the schemas should be from the palette for easier modification, i.e. changing the palette should affect the components. This won't work in this instance.

**Comment by imincheva** [comment]
> > **Dark Mode:**
> > 
> > * Toggle icon should be white 60% in Idle state.
> > * Toggle icon should be white 80% in Hover state.
> > * Toggle icon should be white 60% in Filled state.
> > * Toggle icon should be white 80% in Filled & Hover state.
> > * Toggle icon should be white 20% in Disabled state.
> 
> White? This directly collides with the philosophy we have for the theming that colors listed in the schemas should be from the palette for easier modification, i.e. changing the palette should affect the components. This won't work in this instance.

This has been done on multiple occasions already (probably 90% of all components in dark mode). As far as I know, the rest of the components use contrast colors with opacity. The example here is from the Avatar.

image

**Comment by AnjiManova** [comment]
### Calendar

**Light Mode:** The dates and their states, as do the month and year pickers, look fine.

**Comment by yradoeva** [comment]
**Calendar**

**Dark Mode**

**Date states:**
**1. Selected + Special - Hover / Focused:** border color (::after) should be white 80% (not primary.50)
Pasted Graphic 72
.
--date-selected-special-hover-border-color
.
var (--igx-calendar-date-selected-special-focus-border-

**2. Selected + Special + Current - Hover / Focused:** inner border (::after) should be white 80% (not primary.50)
Pasted Graphic 76
.
--date-selected-special-hover-border-color
.
--date-selected-special-focus-border-color


**Year / Month picker**
**3. Primary and Secondary Title:** text color should be white (not grays.800)
Select Date
.
Pasted Graphic 83

**4. Current - Hover:** text color should stay the same as in idle - grays.900 (not grays.800)
Pasted Graphic 81
.
Pasted Graphic 82

**Comment by yradoeva** [comment]
**Calendar**
**Dark Mode:**
Dates and their states, month and year pickers look good to me.

**Comment by imincheva** [comment]
### Combo
**Dark Mode**
- Clear icon should be white 60% across all states except for disabled (the color in disabled state is correct)

### Select
**Light Mode**
- Toggle color in Idle & Hover and Filled & Hover isn't fixed. It should be grays.700

**Dark Mode**
- Toggle color in Idle & Hover and Filled & Hover isn't fixed. It should be white 80%

**Comment by andiesm813** [comment]
> ### Combo & Select
> * Label should be linked with caption text style
> * Gap between clear icon and toggle icon should be 4px in total on the Combo in Medium size. Currently, both the clear and toggle have 4px padding on both sides, which results in a bigger gap. The same gap should be applied on the Select, but instead between the suffix icon and toggle icon. @andiesm813 can you please specify what gap needs to be applied for Large and Small, because we don't have them in the kit.
> 
> **Light Mode:**
> 
> * Toggle icon should be grays.600 in Idle state. (The same var is used in hover state, but it probably shouldn't be like this, because on hover the toggle should be grays.700 like it's currently implemented.)
> * Toggle icon should be grays.600 in Filled state.
> * Toggle icon should be grays.700 in Filled & Hover state.
> * Toggle icon should be grays.900 20% in Disabled and Filled & Disabled states.
> 
> **Dark Mode:**
> 
> * Toggle icon should be white 60% in Idle state.
> * Toggle icon should be white 80% in Hover state.
> * Toggle icon should be white 60% in Filled state.
> * Toggle icon should be white 80% in Filled & Hover state.
> * Toggle icon should be white 20% in Disabled state.

Hey @imincheva in TREE component we keep the same gap for all 3 sizes. Can't we do the same in this case?
Thanks!

**Comment by imincheva** [comment]
> > ### Combo & Select
> > 
> > * Label should be linked with caption text style
> > * Gap between clear icon and toggle icon should be 4px in total on the Combo in Medium size. Currently, both the clear and toggle have 4px padding on both sides, which results in a bigger gap. The same gap should be applied on the Select, but instead between the suffix icon and toggle icon. @andiesm813 can you please specify what gap needs to be applied for Large and Small, because we don't have them in the kit.
> > 
> > **Light Mode:**
> > 
> > * Toggle icon should be grays.600 in Idle state. (The same var is used in hover state, but it probably shouldn't be like this, because on hover the toggle should be grays.700 like it's currently implemented.)
> > * Toggle icon should be grays.600 in Filled state.
> > * Toggle icon should be grays.700 in Filled & Hover state.
> > * Toggle icon should be grays.900 20% in Disabled and Filled & Disabled states.
> > 
> > **Dark Mode:**
> > 
> > * Toggle icon should be white 60% in Idle state.
> > * Toggle icon should be white 80% in Hover state.
> > * Toggle icon should be white 60% in Filled state.
> > * Toggle icon should be white 80% in Filled & Hover state.
> > * Toggle icon should be white 20% in Disabled state.
> 
> Hey @imincheva in TREE component we keep the same gap for all 3 sizes. Can't we do the same in this case? Thanks!

It's not really up to me to make a decision. As far as I remember, the different gaps have already been communicated with the team, since they've already been implemented and validated months ago by you and our team, although there isn't a handoff, where these are described. Since there's no handoff I can use for reference, I assume this has been communicated in other way, but it can't really be traced back. That's why I'm asking for validation from your side as well, so we can avoid discrepancies. 

If we choose to keep the same gap for all 3 sizes, this would mean that the change has to be logged as a feature request and the input group will require another update. 

The issue here is that in the UI kit the gap in the prefix container of the input control (meaning the space between left prefix icon, prefix text and right prefix icon) is set to 8px, while the gap between the clear icon and the toggle in Combo and the suffix icon and the toggle in Select is set to 4px. Although in the UI kit these are handled in a different way (meaning the architecture of the component in the UI kit is slightly different than the one in the implementation), the gap should stay consistent across all input type controls. 

### In summary, we're left with two questions: 
**- Should the gap be set to 4px or 8px in all input controls in medium size?**
**- Should the gap be the same across the different sizes like you suggested, since it'll require new feature request?**

**Comment by imincheva** [comment]
### Grids

So far I've managed to review the variables list Marin provided us with in light mode, which seems to be implemented correctly. However, the provided list doesn't fully reflect what has already been implemented in the UI kit. I'm not aware how the required changes have been communicated (if possible please reference the issue/feature request/handoff where these are described, so we're all on the same page) and whether it has been mentioned that this variables list doesn't fully cover the required changes, but so far I've encountered many discrepancies in terms of sizing and colors. Some of them I'm going to list here:

- Background color of the selected state is set to grays.200 in the UI kit, while the provided variable in the list says that  --cell-selected-background should be primary.50. Since this color is used across multiple states in the UI kit, this will require update on the already implemented designs in the kit.
- Background color of the active cells in row editing more (not the cell which is being edited, but the ones adjacent to it on the same row) is set to grays.200 in the UI kit, while in the implementation they're set to surface color, since they use --row-odd/even--background and that's the correct color in the provided list
- Color of the excel style filter icon doesn't reflect the one in the UI kit.
- According to the UI kits the cells are supposed to have a height of 48px in large size, while in the implementation it's set to 50px.


If this hasn't been communicated properly, it'll be better for the changes to be implemented before we continue with the review, since the Grids are massive components and there will probably be a lot of comments from our side and these require a lot of effort from our team to trace.

**Comment by imincheva** [comment]
### Grids
As I've mentioned in my previous comment, during the review of the Grids we've encountered a huge number of discrepancies - some related to discrepancies between the provided list with variables from Andie and the UI kit, others from inaccurate implementation. There's definitely some lack of communication, since we've already left over 30 comments in a [branch](https://www.figma.com/design/dkGpSYipbp68uZ1VpRfmsJ/branch/H3K4myCc7MzxWuwIjCsWzS/Indigo.Design-UI-Kit-for-Indigo-1.5-Workfile?m=auto&node-id=13264-114&t=srsd1rbZRcaOCfat-1) in Figma.

The team already agreed that when there's 5 of more discrepancies, we'll stop with the design review and the responsible people for implementation should review their work again. Additionally, our team is not responsible for the design decisions in Indigo and thus can't make further decisions for the current discrepancies and whether the designs in the kit or the ones in the implementation are accurate.

As a result of this situation, we'll stop with the review for now and I hope @andiesm813 and @desig9stein can figure out how to solve the issues. In the branch I linked above are the discrepancies we've managed to find so far, so they can be helpful but please take into consideration that no matter what we've found, **everything should be verified again**, since we covered only a small part of the design.

**Comment by andiesm813** [comment]
> ### Grids
> As I've mentioned in my previous comment, during the review of the Grids we've encountered a huge number of discrepancies - some related to discrepancies between the provided list with variables from Andie and the UI kit, others from inaccurate implementation. There's definitely some lack of communication, since we've already left over 30 comments in a [branch](https://www.figma.com/design/dkGpSYipbp68uZ1VpRfmsJ/branch/H3K4myCc7MzxWuwIjCsWzS/Indigo.Design-UI-Kit-for-Indigo-1.5-Workfile?m=auto&node-id=13264-114&t=srsd1rbZRcaOCfat-1) in Figma.
> 
> The team already agreed that when there's 5 of more discrepancies, we'll stop with the design review and the responsible people for implementation should review their work again. Additionally, our team is not responsible for the design decisions in Indigo and thus can't make further decisions for the current discrepancies and whether the designs in the kit or the ones in the implementation are accurate.
> 
> As a result of this situation, we'll stop with the review for now and I hope @andiesm813 and @desig9stein can figure out how to solve the issues. In the branch I linked above are the discrepancies we've managed to find so far, so they can be helpful but please take into consideration that no matter what we've found, **everything should be verified again**, since we covered only a small part of the design.

@imincheva so there are discrepancies among the list of variables i provided and the ui kit? is that the issue? Because, like you said before, all the variables i provided in that list are **perfectly implemented**....  So if the issue is that the ui kit does not reflect what i provided in the variables list, i guess i didnt understand that from the previous messages.

**Comment by sdimchevski** [comment]
@andiesm813 Yes, that's the issue. When you say the variables are perfectly implemented from that list I guess you've planned to make a branch so we can have everything perfectly aligned in the UI kit, right?

**Comment by andiesm813** [comment]
> @andiesm813 Yes, that's the issue. When you say the variables are perfectly implemented from that list I guess you've planned to make a branch so we can have everything perfectly aligned in the UI kit, right?

Ok, i'll compare my list of variables towards what's in the UI kit and figure out what's not aligned @sdimchevski

**Comment by imincheva** [comment]
@simeonoff The paddings on the searchbar in the combo dropdown has incorrect left and right paddings. They should be 8px. The size of the input should also correspond to medium size.

**Comment by desig9stein** [comment]
@imincheva @andiesm813, You can test the grid now, this is the branch you should use for the theming package `mpopov/indigo-grids--10.1.x`, if you find more than 2/3 mistakes, stop testing and report back to me. Keep in mind that icons and icon sizes will be introduced in another PR.

**Comment by imincheva** [comment]
### Grids
- Cell summary cells' height across all sizes doesn't correspond to the ones in the UI kit
- In pivot grid part of a border is missing. This is relevant for all themes, so it should probably be logged as a bug
image

- In pivot grid the padding on igx-pivot-row-dimension-header should be 6px, not 4px
image

- In GroupBy the column name and text labels should be linked to Detail 2 and Detail 1 respectively.
image
image

- The container on the image from row editing banner in small size should be 24px, not 24.5px. In the rest of the sizes it's okay, that's why I'm mentioning it here.
image

- Size of the Select in Paginator should correspond to the overall size of the grid.
- Input size here on Grid  should correspond to the size of the grid.
image

- The element which appears when you drag a header is 1px higher in all three sizes. The correct heights are 48, 40, 32 for large, medium and small respectively
- This part of the filter doesn't correspond to the design at all - paddings, margins, sizes, colors, etc.
image

- Top padding on this part should be 8px in all sizes
image

- Bottom padding here should be 4px in medium and small size. Already done in small.
image

- There should be 4px margin/gap between this title and the buttons bellow in large and medium size
image

- List paddings and gap is missing in all sizes
image



<hr>

**Light Mode**
- Background of the disabled button should be white. Currently it's set to transparent. The gap between the header and the buttons should also be 4px in large and medium size.
image

- --group-row-background should be surface, not grays.100.
- background color of the toolbar should be surface, not transparent
- border color of the toolbar should be grays.400, currently it's grays.900 30%
- background color of the paginator should be surface, currently it's transparent
- border color of the paginator should be grays.400, currently it's grays.900 30%
- Elevation 5 applied on --drag-shadow doesn't look like in the kit, probably should be updated

**Dark Mode**
PENDING

**Comment by imincheva** [comment]
- Bottom padding on the Title should be 12px in medium and 8px in small
image

- Bottom margin below the buttons on the image should be added: 8px in large and medium and 4px in small
image

**Comment by yradoeva** [comment]
**Grid**
**Excel Style Filter - light & dark theme**

As far as I understood, the top padding for the list will be 16 px in all sizes? So, it should be implemented in S, as it is currently 8px there. 
image
. 
image

File: projects/igniteui-angular/src/lib/core/styles/components/grid/_excel-filtering-theme.scss [hunk 1]
```diff
@@ -48,7 +48,7 @@
         flex-grow: 1;
 
         @if $variant == 'indigo' {
-            box-shadow: elevation(3), 0 0 0 rem(1px) color(null, 'gray', if($theme-variant == 'light', 400, 100));
+            box-shadow: elevation(if($theme-variant == 'light', 3, 2)), 0 0 0 rem(1px) color(null, 'gray', if($theme-variant == 'light', 400, 100));
```
Comment (simeonoff) [inline projects/igniteui-angular/src/lib/core/styles/components/grid/_excel-filtering-theme.scss:51]: This should ideally come from the schemas, the same way it's set for the drop-down component schema.

File: projects/igniteui-angular/src/lib/core/styles/components/grid/_excel-filtering-theme.scss [hunk 2]
```diff
@@ -399,12 +404,39 @@
...
         &:focus {
-            background: color($color: 'gray', $variant: 100);
             color: var-get($theme, 'excel-filtering-actions-hover-foreground');
+
+            @if $variant == 'indigo' {
+                @if $theme-variant == 'light' {
+                    background: color($color: 'gray', $variant: 200);
+                } @else {
+                    background: contrast-color($color: 'gray', $variant: 50, $opacity: .1);
+                }
```
Comment (simeonoff) [inline projects/igniteui-angular/src/lib/core/styles/components/grid/_excel-filtering-theme.scss:414]: Again, this ideally comes from the schema, not something that is hardcoded here.

**Review by simeonoff — COMMENTED** [review]
I saw a few instances of hardcoded colors based on the theme variant (light/dark). Can we have them in the schema declarations?

File: projects/igniteui-angular/src/lib/core/styles/components/grid/_excel-filtering-theme.scss (hunk 1 above)
Comment (desig9stein) [reply projects/igniteui-angular/src/lib/core/styles/components/grid/_excel-filtering-theme.scss:51]: I can't do anything unless we want to expose new variables for all themes.

**Review by SisIvanova — APPROVED** [review]
